tasks/fullStack/app/controllers/reviews_controller.rb

```diff
@@ -1,29 +1,45 @@
 class ReviewsController < ApplicationController
+  before_action :set_product, only: [:new]
 
   DEFAULT_TAGS = ['default']
 
   def index
-    if params[:shop_id].present? && Shop.where("id = #{params[:shop_id]}").present?
+    params[:shop_id] ||= 1
+    @shop = Shop.find(params[:shop_id])
+
+    if @shop.present?
       params[:per_page] ||= 10
       offset = params[:page].to_i * params[:per_page]
 
+      @shops = Shop.where.not(id: params[:shop_id]).limit(5)
```

Comment:
What's the purpose for making this extra query?

Reply:
That query is used to fetch the shops that are displayed within the filter dropdown

Reply:
👍 sorry, my bad, completely forgot the filter dropdown